auth/src/main/java/com/vimeo/networking2/Authenticator.kt

```diff
@@ -247,6 +247,51 @@ interface Authenticator {
         callback: VimeoCallback<VimeoAccount>
     ): VimeoRequest
 
+    /**
+     * Find a supported SSO domain that matches the [domain] parameter.
+     *
+     * @param domain A domain, also known as hostname, that might be supported for SSO by the Vimeo API.
+     * @param callback Callback to be notified of the result of the request.
+     *
+     * @return A [VimeoRequest] object to cancel API requests.
+     */
+    @Internal
+    fun fetchSsoDomain(
+        domain: String,
+        callback: VimeoCallback<SsoDomain>
+    ): VimeoRequest
+
+    /**
+     * Obtain the URI which can be used to log in the user and receive back a URI that can be exchanged using
+     * [authenticateWithSso] for a logged in account. Obtain the [SsoDomain] required by this function first using
+     * [fetchSsoDomain].
+     *
+     * @param ssoDomain The domain info obtained from the API that specifies the [SsoDomain.connectUrl] which will be
+     * used as the base for this URI.
+     * @param responseCode The response code that can be used to identify the origin of the redirect URI.
+     *
+     * @return The URI which can be opened in a browser.
```

Comment:
For this comment and the next one on documentation, I opted to copy the docs from the `authenticateWithCodeGrant` instead of just changing the return doc. We worked a lot on the code grant docs in a previous PR, and SSO follows the same flow, so we can just reuse them on these functions and slightly adapt them for SSO.